resources/views/backend/posts/show.blade.php

```diff
@@ -54,7 +55,7 @@
               <div style="padding-top: 10px;"><b>{{ __('posts.content') }}</b></div>
               <p>Lorem ipsum represents a long-held tradition for designers, typographers and the like. Some people hate it and argue for its demise, but others ignore the hate as they create awesome tools to help create filler text for everyone from bacon lovers to Charlie Sheen fans.</p>
             </ul>
-            <a href="#" class="btn btn-danger btn-block btn-flat">
+            <a href="#" class="btn btn-danger btn-block btn-flat btn-delete-item" data-title="{{ __('Confirm deletion!') }}" data-confirm="{{ __('Are you sure you want to delete?') }}">
```

Comment:
sao không đưa vào file lang luôn em ơi.
`Confirm deletion!`, `Are you sure you want to delete?`

Reply:
dạ để em xem lại ạ